index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="/style.css">
```

Comment:
I think it is better to use relative paths.
```<link rel="stylesheet" href="./style.css">```
instead of
```<link rel="stylesheet" href="/style.css">```

Reply:
> I think it is better to use relative paths. `<link rel="stylesheet" href="./style.css">` instead of `<link rel="stylesheet" href="/style.css">`

 

- [ ] It is always important to add the WHY, this is for the student to know the reason you are requesting changes.

Reply:
Oh, ok, I'll keep it in mind in the future. It is good to have relative file paths because we don't know the server's directory structure where the site will be hosted most of the time. So using `/style.css` will find the file in the root folder, whereas `./style.css` will find the file in the same folder as the HTML file.

In short words, the styles will not break on servers.